linkatos/activities.py

```diff
@@ -11,34 +11,40 @@ def is_empty(events):
 def is_url(url_cache):
     return url_cache is not None
 
-def event_consumer(expecting_url, url_cache, slack_client,
-                   fb_credentials, firebase):
 
+def remove_url_from(url_cache_list, index):
+    url_cache_list.pop(index)
```

Comment:
I thought about it, but forgot about it.
Would you agree that it's easier to read with the function?